pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.black]
+line-length = 120
+target-version = ["py36", "py37", "py38"]
+extend-exclude = "robosuite/((models/assets)|(controllers/config))"
+
+[tool.isort]
+profile = "black"
+line_length = 120
+skip=["__init__.py"]
+py_version = "all"
```

Comment:
we could add the line `filter_files = true`
https://pycqa.github.io/isort/docs/configuration/options.html#filter-files
This will apply the file filters even if a user passes the files manually, e.g., `isort *`

Reply:
Done.